Volume1

mariannebautista opened the pull request.

taylomol000 committed:
General Prep

taylomol000 committed:
Forgot to save one change

mariannebautista committed:
Noted my entries (1-28)

Dhreetii committed:
Noted Dhreeti's part

Dhreetii committed:
Tagged entries on lines 226-270 (with some incomplete)

mariannebautista committed:
Updates to entries 1-28

mariannebautista committed:
More updates to entries 1-28

mariannebautista committed:
Updated entries 1-28

mariannebautista committed:
Entries 1-28 updates

mariannebautista committed:
Updated entries 1-28

mariannebautista committed:
updated entries 1-28

mariannebautista committed:
Finishing first run-through of 1-28

ketchley committed:
Update volume1.xml

Dhreetii committed:
updated my entries

Dhreetii committed:
Merge branch 'volume1' of https://github.com/eba-diary/tagged-volumes into volume1

Dhreetii committed:
updated entry

mariannebautista committed:
Added some persName tags

Dhreetii committed:
updates to my entries

Dhreetii committed:
Merge branch 'volume1' of https://github.com/eba-diary/tagged-volumes into volume1

Dhreetii committed:
updates to my entries

Dhreetii committed:
updates to my entries

Dhreetii committed:
added persName - Fatima

Thurst2165 committed:
Update volume1.xml

Thurst2165 committed:
Update volume1.xml

Thurst2165 committed:
Merge branch 'volume1' of https://github.com/eba-diary/tagged-volumes into volume1

Dhreetii committed:
fixed some random tags, changed grand duke tag

Dhreetii committed:
marked my new entries

ketchley committed:
Update volume1.xml

edit tags

ketchley committed:
Update volume1.xml

ketchley committed:
Update volume1.xml

ketchley committed:
Update volume1.xml

ketchley committed:
Update volume1.xml

Thurst2165 committed:
Update volume1.xml

Dhreetii committed:
tagging updates

mariannebautista committed:
edits to part 4

Dhreetii committed:
updates vol 1

Dhreetii committed:
Merge branch 'volume1' of https://github.com/eba-diary/tagged-volumes into volume1

Dhreetii committed:
updates - paintings

Dhreetii committed:
fixed one name

mariannebautista committed:
updates-lastsection

mariannebautista committed:
more updates last section

mariannebautista committed:
finishing last section

Dhreetii committed:
updates

Dhreetii committed:
Merge branch 'volume1' of https://github.com/eba-diary/tagged-volumes into volume1

Thurst2165 committed:
Update volume1.xml

Thurston section completed (unrevised)

Dhreetii committed:
update dates

Dhreetii committed:
updated all dates

ketchley committed:
Update volume1.xml

first run of XSLT edits

ketchley committed:
Update volume1.xml

ketchley committed:
Update volume1.xml

ketchley committed:
Update volume1.xml

emmafritzberg commented on volume1.xml:
```diff
@@ -0,0 +1,749 @@
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
```
@emmafritzberg move into diary-volumes

emmafritzberg replied:
Move to : https://github.com/eba-diary/apps/tree/main/emmabandrews/data/volumes

Dhreetii committed:
update persNames

Dhreetii committed:
Merge branch 'volume1' of https://github.com/eba-diary/tagged-volumes into volume1

mariannebautista committed:
placenamechanges

mariannebautista committed:
placenamesupdate2

ketchley committed:
Update volume1.xml

Thurst2165 committed:
Update volume1.xml

ketchley committed:
Update volume1.xml

update header

ketchley committed:
Update volume1.xml

Dhreetii committed:
name updates

Dhreetii committed:
updates to hotels, boats, placeNames (unfinished)

Dhreetii committed:
placeName updates

ketchley committed:
Update volume1.xml

Dhreetii committed:
minor tag and comment fixes

Dhreetii committed:
Merge branch 'volume1' of https://github.com/eba-diary/tagged-volumes into volume1

Thurst2165 committed:
Update volume1.xml

Primarily fixing and updating some tags/comments in Thurston's section (feb 18 1890 - March 21 1890). placeName -> historical tag corrections

ketchley committed:
Update volume1.xml

ketchley committed:
Update volume1.xml

ketchley committed:
Update volume1.xml

Thurst2165 committed:
Update volume1.xml

Fixed the highlighted comments in XLT file.

Dhreetii committed:
Fixed most written comments

Dhreetii committed:
placeName to historical updates

Thurst2165 committed:
Update volume1.xml

Final fixes to placeName tags -> name type="historical" and fixes to all remaining comments

Thurst2165 committed:
Update volume1.xml

Fixed all apparent errors of persName ref="..." type="historical" -> persName type="historical" ref="..."
